### PR TITLE
Include scope variables

### PR DESCRIPTION
**Pull request by dvd101x**
Hi, Jos

This is only WIP as it currently does not track the current scope, just the empty scope at the beginning.

### CODE REVIEW

**Comment by dvd101x** [comment]
Hi, this PR now solves two issues. It autocompletes variables in scope dynamically (not only the initial state) but also everything related to mathjs is autocompleted dynamically (not only related to the initial mathjs state).

**Comment by josdejong** [comment]
O that sounds promising. I'll have a look coming Wednesday

File: src/mathjs-lang.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  *
  * @param {Object} math A mathjs instance
```
Comment (josdejong) [inline src/mathjs-lang.js]: Can you update the JSDoc so it matches the new `getMath` and `getScope` arguments?

**Review by josdejong — APPROVED** [review]
Nice 😎 , this looks good! I made one small inline comment, can you have a look at that?

**Comment by josdejong** [comment]
O, and one more remark: we can remove the Todo list from the readme, its finished 🎉

File: src/mathjs-lang.js (hunk 1 above)
Comment (dvd101x) [reply src/mathjs-lang.js]: yes of curse, thanks!

**Comment by josdejong** [comment]
Nice! I'll merge your PR now (next time, feel free to merge it yourself after I approved it).

**Pull request merged**